app/src/main/java/appricottsoftware/clarity/RegisterActivity.java

```diff
@@ -63,65 +62,52 @@ public void onClick(View v) {
                     return;
                 }
 
-                // create instance of clarityClient
-                // pass information to database to store
-                if (seeSurvey == true){
-                    Intent surveyActivityIntent = new Intent(this, SurveyActivity.class);
-                    surveyActivityIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK|Intent.FLAG_ACTIVITY_CLEAR_TOP);
-                    startActivity(surveyActivityIntent);
-                    finish();
-                }
-
-                else {
-                    ClarityApp.getRestClient().registerRequest(emailString, hashedPassword, this, new JsonHttpResponseHandler() {
-                        @Override
-                        public void onSuccess(int statusCode, Header[] headers, JSONObject response) {
-                            Log.e(TAG, "onSuccess1 : " + response.toString() );
-                            super.onSuccess(statusCode, headers, response);
-
-                            // Setting userID for the session from returned JSON object
-                            try {
-                                ClarityApp.getSession(getApplicationContext()).setUserID(response.getInt("userId"));
-
-                                // After successful save of user info on back end
-                                // Switch to home activity
-                                Intent homeActivityIntent = new Intent(getApplicationContext(), HomeActivity.class);
-                                homeActivityIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TOP);
-                                homeActivityIntent.putExtra("loginType", getString(R.string.registered_login_type));
-                                startActivity(homeActivityIntent);
-                                finish();
-                            }
-                            catch (JSONException e) {
-                                e.printStackTrace();
-                            }
+                ClarityApp.getRestClient().registerRequest(emailString, hashedPassword, this, new JsonHttpResponseHandler() {
+                    @Override
+                    public void onSuccess(int statusCode, Header[] headers, JSONObject response) {
+                        Log.e(TAG, "onSuccess1 : " + response.toString() );
```

Comment:
Is this onSuccess response an error? If not, consider using Log.v.

Reply:
Removed.